rotkehlchen/accounting/history_base_entries.py

```diff
@@ -114,6 +114,20 @@ def make_default_accounting_settings(pot: 'AccountingPot') -> dict[str, BaseEven
         count_cost_basis_pnl=False,
         method='acquisition',
     )
+    staking_deposit_key = str(HistoryEventType.STAKING) + '__' + str(HistoryEventSubType.DEPOSIT_ASSET)  # noqa: E501
+    result[staking_deposit_key] = BaseEventSettings(
+        taxable=False,
+        count_entire_amount_spend=True,
```

Comment:
This sounds wrong. Depositing something from staking should just count as transfer. Nothing should be spent, and no PNL should be counted. Same below.